data/core/common.lua

```diff
@@ -869,4 +869,69 @@ function common.rm(path, recursively)
 end
 
 
+---Open the given resource using the system launcher.
+---The resource can be an url, file or directory in most cases...
+---@param resource string
+---@return boolean success
+function common.open_in_system(resource)
+  -- Detect platforms
+  local function detect_platform()
+    if PLATFORM:lower():find("unknown", 1, true) then
+      -- AmigaOS: ENV: is a system assign
+      local f = io.open("ENV:", "r")
```

Comment:
This is a device and not a file. I am not sure if the `io.open()` will return anything valuable